Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -88,6 +91,20 @@ - (void)setPlaceholderColor:(UIColor *)placeholderColor
   _placeholderView.textColor = _placeholderColor ?: defaultPlaceholderColor();
 }
 
+- (void)setReactTextAttributes:(RCTTextAttributes *)reactTextAttributes
+{
+  if (!reactTextAttributes.effectiveTextAttributes || [reactTextAttributes isEqual:_reactTextAttributes]) {
```

Comment:
Can you add a variable for `effectiveTextAttributes` to avoid computing it twice, looks like a pretty big getter https://github.com/facebook/react-native/blob/9d16ddf639056b2fc3ce2194e20ab658bcc0397b/Libraries/Text/RCTTextAttributes.m#L82 (I hate properties :))

Reply:
@janicduplessis 👍 I removed `!reactTextAttributes.effectiveTextAttributes` check because it cannot be `nil`.